src/main/kotlin/info/novatec/docu/parser/element/CallActivityParser.kt

```diff
@@ -0,0 +1,31 @@
+package info.novatec.docu.parser.element
+
+import info.novatec.cbdg.models.elements.CallActivity
+import info.novatec.constants.BPMN_ATTRIBUTE_CALLED_ELEMENT
+import info.novatec.constants.BPMN_TAG_CALL_ACTIVITY
+import org.camunda.bpm.model.bpmn.instance.Process
+
+object CallActivityParser {
+
+    /**
+     * Extracts all call activities from a process, maps them to a [CallActivity] and puts them in a list.
+     *
+     * @param process The process from which all CallActivities are extracted to a [CallActivity] object and put in the returned list
+     *
+     * @return the list of CallActivities as [CallActivity] object with the information about Id, Name and the Called Element and Element Documentation,
+     * comprised in one object.
+     */
+    fun parseToCallActivityList(process: Process): List<CallActivity> {
+        val callActivityFlowElementList = process.flowElements.filter { it.elementType.typeName.equals(BPMN_TAG_CALL_ACTIVITY) }
```

Comment:
Kotlin allows [operator overloading](https://kotlinlang.org/docs/operator-overloading.html), so you can use == instead of equals() to compare objects because the == is overloaded with a call to equals.
This provides the additional benefit of null safety since according to the documentation == is always true for null == null.

Doesnt necessarily need to be changed, but its nice to know.